Sources/MockoloFramework/Models/ClassModel.swift

```diff
@@ -46,7 +46,8 @@ final class ClassModel: Model {
         self.needInit = needInit
         self.initParams = initParams
         self.offset = offset
-        self.attribute = Set(attributes).joined(separator: " ")
+        let ret = attributes.filter {$0.contains(String.available)}
```

Comment:
This has been bothering me for a while; what is `ret`? "Return?" Can we use a more descriptive name like `availableAttributes`?

Reply:
yes